core/src/main/scala/stryker4s/config/Thresholds.scala

```diff
@@ -1,6 +1,17 @@
 package stryker4s.config
+import stryker4s.extensions.exceptions.InvalidThresholdValueException
 
-case class Thresholds(// high: Int = 80, // TODO: Not supported yet
-                      // low: Int = 60,  // TODO: Not supported yet
-                      break: Int) {
+case class Thresholds(high: Int = 80, low: Int = 60, break: Int = 0){
+  failIfNotPercentage(high, low, break)
+  if(high <= low)
```

Comment:
Not sure how Stryker does it. But I'm unsure about these being `<=`. I think it's fairly intuitive to put both at the same, if you don't care about the lower end of the two.

Reply:
You're correct, it shouldn't be `<=`. Stryker does allow 'low' to equal 'high'.